app/controllers/application_controller.rb

```diff
@@ -2,8 +2,16 @@
 
 class ApplicationController < ActionController::API
   include DeviseTokenAuth::Concerns::SetUserByToken
+  include Pundit
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :authenticate_user!
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+
+  private
+
+  def user_not_authorized
+    render status: :forbidden
```

Comment:
I had in mind this:
```suggestion
    render json: { error: 'You are not authorized to perform this action.' }, status: :forbidden
```

Reply:
It would be best to have both error and 403 as HTTP Code as Wojtek is proposing